recipes/opencolorio/all/conanfile.py

```diff
@@ -42,13 +42,17 @@ def configure(self):
         if self.options.shared:
             del self.options.fPIC
 
-        if self.settings.compiler.cppstd:
-            tools.check_min_cppstd(self, "11")
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 11)
+        if not self.options["tinyxml"].with_stl:
+            raise ConanInvalidConfiguration("tinyxml option 'with_stl' has to be enabled to build opencolorio!")
```

Comment:
So it can't work out of the box currently because stl is False by default in tinyxml.

Reply:
I now fixed the whole tinyxml issue by patching OCIO work with `with_stl` both on and off. Hence the option check was removed.